h/static/scripts/controllers/tooltip-controller.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/**
+ * A custom tooltip similar to the one used in Google Docs which appears
+ * instantly when activated on a target element.
+ *
+ * The tooltip is displayed and hidden by setting its target element.
+ *
+ *  var tooltip = new Tooltip(document.body);
+ *  tooltip.setState({target: aWidget}); // Show tooltip
+ *  tooltip.setState({target: null}); // Hide tooltip
+ *
+ * The tooltip's label is derived from the target element's 'aria-label'
+ * attribute.
+ *
+ * @param {Element} el - The container for the tooltip.
+ */
+function TooltipController(el) {
+  var self = this;
+
+  el.addEventListener('mouseover', function () {
```

Comment:
I assumed that this wouldn't work on mobile but it turns out that iOS Safari at least does synthesize `mouseover` and `mouseout` events when the element is clicked.